BranchSDK/BNCPreferenceHelper.m

```diff
@@ -817,23 +803,6 @@ - (void) setInvokeRegisterApp:(BOOL)invoke {
     }
 }
 
-- (BOOL) useEUServers {
```

Comment:
@echo-branch Is this property not persistent?

Reply:
I was thinking it would be set on app launch, either in code or config file.